packages/web/src/features/translation/verse-utils.ts

```diff
@@ -130,7 +130,7 @@ export function incrementVerseId(verseId: string) {
  * @returns The translated name of the book.
  */
 export function bookName(bookId: number, t: TFunction) {
-  return t(bookKeys[bookId - 1], { ns: 'bible' });
+  return t(bookKeys[bookId - 1].toLowerCase(), { ns: 'bible' });
```

Comment:
I decided it was simpler to capitalize the keys in `bookKeys` and then lowercase them here, rather than trying to capitalize them in the API logic. The capitalization logic gets a little more complicated when you're wanting to convert from `1ch` to `1Ch`.

This version of `bookKeys` is also more accurate, because it matches the filenames in `data/morphology`, AND the JS gloss filenames (for example: `https://hebrewgreekbible.online/EnglishGlosses/1ChGloss.js`)

Reply:
I think this is a good approach. It might be good to match the keys in bible.json to the same format so they are all the same.

Reply:
I thought about that, but our current convention is to have all lowercase translation keys. I'd rather keep it that way.